### ISSUE TITLE
Unable to navigate to a community from a post with VoiceOver

### ISSUE THREAD

**Issue by WestonHanners** [buresdv/Mlem#184]
**Describe the bug**
Unable to navigate, activating a community header opens the post

**To Reproduce**
Try to navigate to a community via a post.

**Mlem Information:**
0.0.7

**Comment by WestonHanners** [comment]
my PR too fix this created a bug with navigation link. I will revisit this once we have improved navigation stack

**Comment by WestonHanners** [comment]
This fix requires some changes to the NavigationStack (thanks Apple)
